.github/workflows/future-release-branch-labeler.yml

```diff
@@ -0,0 +1,43 @@
+name: Future Release Branch Pull Request Labeler
+
+on:
+  pull_request_target:
+    branches:
+      - dev-[0-9].[0-9][0-9]
+    types:
+      - opened
+      - reopened
+
+jobs:
+  add-hold-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+          curl --request PATCH \
+          --url ${{ github.event.pull_request.issue_url }} \
+          --header 'Authorization: token ${{ secrets.GITHUB_ACTIONS_TOKEN }}' \
+          --header 'Accept: application/vnd.github.v3+json' \
+          --data '{"labels": ["hold"]}'
+  add-milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+          # grab the version using the branch of the base repository
+          milestone=${GITHUB_BASE_REF:4}
+
+          # grab the number associated with milestone not the name of milestone
+          curl --request GET \
```

Comment:
Does the milestone number always correspond to the branch?
milestone = `1.20`
branch = `dev-1.20`

Reply:
OK. Is the `GET` request necessary?

Reply:
Yes, when setting the milestone, you can't pass in the title/name of the milestone (ex. `1.20`), instead, you need the number associated with that title (ex. `1.20` number is 46 -- it also shows in the URL: https://github.com/kubernetes/website/milestone/46)

Therefore, I have to get the list of milestones and find the one that matches the title (1.20) to get the number. 

I wasn't able to find a better way

Reply:
OK. I thought perhaps you could access the information from the site's configuration (config.toml) in the
`dev-*` branch instead of making another API call.